readme.md

```diff
@@ -215,6 +215,31 @@ const audioTrackUrl = 'https://test-audio.netlify.com/Various%20Artists%20-%2020
 })();
 ```
 
+Or use [`@tokenizer/s3`](https://github.com/Borewit/tokenizer-s3) to determine the file type of a file stored on [Amazon S3](https://aws.amazon.com/s3):
+
+```js
+const FileType = require('file-type');
+const S3 = require('aws-sdk/clients/s3');
+const {makeTokenizer} = require('@tokenizer/s3');
+
+(async () => {
+	// Initialize the S3 client
+	const s3 = new S3();
+
+	// Initialize the S3 tokenizer.
+	const s3Tokenizer = await makeTokenizer(s3, {
+		Bucket: 'affectlab',
```

Comment:
Why are these PascalCase? `Bucket` => `bucket`, if you're taking API feedback.

Reply:
That is inherited from the Amazon S3 client API.

Reply:
Ah ok